### DISCUSSION TITLE
Web interface - no recordings listed

### DISCUSSION THREAD

**Discussion by petricf** [Motion-Project/motionplus#100] [Q&A] [answered]
Installed motionplus deb v0.1.1 on debian 12 (bookworm).

Problem: Webcontrol interface - try to view recordings.

There are recordings listed in the footage directory. They have also been recorded in the mysql database. But when i select "Recordings" -> "\<camera>" nothing happens. Even no headline of the recordings list is displayed.

A restart of the service does not help.

The live preview and full size view are working properly.

The footage directories are owned by motion:motion.

Below the log part (debug level 8) of the web interface. When selecting the recording menu (and the specific camera) nothing is logged.

    [DBG][STR][00:wc00] webu_parseurl: Sent url: /
    [DBG][STR][00:wc00] webu_parseurl: Decoded url: /
    [DBG][STR][00:wc00] webu_parms_edit: camid: >< thread: >-1< cmd1: >< cmd2: >< cmd3: ><
    [DBG][ALL][00:wc00] webu_hostname: Full Host:  //192.168.2.102:8080
    [INF][ALL][00:wc00] webu_client_connect: Connection from: 192.168.2.20
    [DBG][ALL][00:wc00] webu_hostname: Full Host:  //192.168.2.102:8080
    [DBG][STR][00:wc00] webu_answer_get: processing get
    [DBG][STR][00:wc00] webu_parseurl: Sent url: /0/config.json
    [DBG][STR][00:wc00] webu_parseurl: Decoded url: /0/config.json
    [DBG][STR][00:wc00] webu_parms_edit: camid: >0< thread: >-1< cmd1: >config.json< cmd2: >< cmd3: ><
    [DBG][ALL][00:wc00] webu_hostname: Full Host:  //192.168.2.102:8080
    [DBG][ALL][00:wc00] webu_hostname: Above message repeats 1 times
    [DBG][STR][00:wc00] webu_answer_get: processing get
    [DBG][STR][00:wc00] webu_parseurl: Sent url: /favicon.ico
    [DBG][STR][00:wc00] webu_parseurl: Decoded url: /favicon.ico
    [DBG][STR][00:wc00] webu_parms_edit: camid: >< thread: >-1< cmd1: >< cmd2: >< cmd3: ><
    [DBG][STR][00:wc00] webu_parseurl: Sent url: /101/mjpg/stream
    [DBG][STR][00:wc00] webu_parseurl: Decoded url: /101/mjpg/stream
    [DBG][STR][00:wc00] webu_parms_edit: camid: >101< thread: >0< cmd1: >mjpg< cmd2: >stream< cmd3: ><
    [DBG][ALL][00:wc00] webu_hostname: Full Host:  //192.168.2.102:8080
    [DBG][ALL][00:wc00] webu_hostname: Above message repeats 1 times
    [DBG][STR][00:wc00] webu_answer_get: processing get
    [DBG][STR][00:wc00] webu_parseurl: Sent url: /102/mjpg/stream
    [DBG][STR][00:wc00] webu_parseurl: Decoded url: /102/mjpg/stream
    [DBG][STR][00:wc00] webu_parms_edit: camid: >102< thread: >1< cmd1: >mjpg< cmd2: >stream< cmd3: ><
    [DBG][ALL][00:wc00] webu_hostname: Full Host:  //192.168.2.102:8080
    [DBG][ALL][00:wc00] webu_hostname: Above message repeats 1 times
    [DBG][STR][00:wc00] webu_answer_get: processing get

Used webclient is firefox (Linux).

Any idea what is going wrong ?

**Comment by Mr-Dave** [comment, marked as answer]
My guess is that it is issue 02 on this [enhancement list](https://github.com/Motion-Project/motionplus/discussions/56).  Try to click on a camera to show it, then go back to click on recordings.

**Reply by petricf** [reply]
You are right. This procedure works.

Hopefully it will be fixed in the next release.

Thank for your support.

**Reply by Mr-Dave** [reply]
FYI:  A fix for this was just pushed to the master branch

**Comment by Prhewett** [comment]
Hi There.
I seem to be having the same problem with the web interface - no video are showing under recordings. In my case selecitng the camera does NOT resolve the problem.
I am using version 0.2.1. The log file doesnt seem to show anything very helpful.
My video files are stored on a samba share but the motionplus app is runing as root under Sudo and has read and write permiossoin to the video files. 
I can open the file manually - jus not through the Motionplus web interface..
Here's a gre of the log:(lvl 8)
peterhewett@pi4:~ $ tail -f /var/log/motionplus.log |grep web
Aug 20 14:40:14 [DBG][STR][00:wc00] webu_parseurl: Sent url: /101/mjpg/stream
Aug 20 14:40:14 [DBG][STR][00:wc00] webu_parseurl: Decoded url: /101/mjpg/stream
Aug 20 14:40:14 [DBG][STR][00:wc00] webu_parms_edit: camid: >101< thread: >0< cmd1: >mjpg< cmd2: >stream< cmd3: ><
Aug 20 14:40:14 [DBG][ALL][00:wc00] webu_hostname: Full Host:  //192.168.86.44:8080
Aug 20 14:40:14 [DBG][ALL][00:wc00] webu_hostname: Above message repeats 1 times
Aug 20 14:40:14 [DBG][STR][00:wc00] webu_answer_get: processing get
Aug 20 14:40:22 [DBG][STR][00:wc00] webu_parseurl: Sent url: /101/movies.json
Aug 20 14:40:22 [DBG][STR][00:wc00] webu_parseurl: Decoded url: /101/movies.json
Aug 20 14:40:22 [DBG][STR][00:wc00] webu_parms_edit: camid: >101< thread: >0< cmd1: >movies.json< cmd2: >< cmd3: ><
Aug 20 14:40:22 [DBG][ALL][00:wc00] webu_hostname: Full Host:  //192.168.86.44:8080
Aug 20 14:40:22 [DBG][ALL][00:wc00] webu_hostname: Above message repeats 1 times
Aug 20 14:40:22 [DBG][STR][00:wc00] webu_answer_get: processing get
Aug 20 14:40:22 [INF][STR][00:wc00] webu_mhd_deinit: Closing connection
Aug 20 14:40:24 [INF][STR][00:wc00] webu_mhd_deinit: Closing connection

**Reply by Mr-Dave** [reply]
Do you see the log messages about the database starting OK and after each event about SQL queries being executed successfully?  (i.e. The most likely reason is that the database hasn't been set up or isn't accepting/providing the list of movies)
